pkg/pillar/dpcmanager/dpc.go

```diff
@@ -176,12 +182,12 @@ func (m *DpcManager) insertDPC(dpc types.DevicePortConfig) {
 }
 
 // Remove by matching TimePriority and Key
-func (m *DpcManager) removeDPC(dpc types.DevicePortConfig) {
+func (m *DpcManager) removeDPC(dpc types.DevicePortConfig, keyOnly bool) {
```

Comment:
I think it would be more clear to introduce a separate removeDPCByKey(key string) function.